functions/openhab.bash

```diff
@@ -178,6 +178,15 @@ openhab_shell_interfaces() {
   fi
 }
 
+## Function to clean the cache for openHAB
+##
+##    openhab_clean_cache()
+##
+openhab_clean_cache() {
+  echo -n "$(timestamp) [openHABian] Cleaning the openHAB cache... "
+  if cond_redirect openhab-cli clean-cache; then echo "OK"; else echo "FAILED"; return 1; fi
```

Comment:
did you test that it works without that it requires input ? IIRC it's asking for confirmation, or is that only when run interactively ?

Reply:
nope, ill look into that